projects/plugins/debug-helper/modules/class-xmlrpc-logger.php

```diff
@@ -235,14 +235,14 @@ public function pretty_print_xml( $xml ) {
 	public function convert_xml_rpc_to_json( $xml ) {
 		// Convert SimpleXML object to an array
 		// phpcs:ignore WordPress.WP.AlternativeFunctions.json_encode_json_encode, WordPress.WP.AlternativeFunctions.json_decode_json_decode
-		$array = json_decode( json_encode( (array) $xml ), true );
+		$array = json_decode( json_encode( (array) $xml, JSON_UNESCAPED_SLASHES ), true );
```

Comment:
One of those encode-decode situations where we may as well to `JSON_UNESCAPED_UNICODE` too to save a few nanoseconds.